mysql/infoschema.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"github.com/cloudspannerecosystem/harbourbridge/internal"
 	"github.com/cloudspannerecosystem/harbourbridge/schema"
+	spangres "github.com/cloudspannerecosystem/harbourbridge/spangres"
```

Comment:
Nit: I am not sure but I think using spangres alias won't effect here since the module name is spangres already. Ignore if that's not the case.